frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/basicTicketImage/basicTicketImage.styles.ts

```diff
@@ -40,9 +30,19 @@ export const Container = styled.div`
 
 export const PropertyName = styled(Typography).attrs({
```

Comment:
I think we should use `<InputLabel />` (to stuck with materialuis family of components for inputs) here and in the pin details, changing the style to adjust to George's design.
Notice that `<InputLabel />` deals with the required prop.